engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailAggregator.java

```diff
@@ -101,16 +101,26 @@ public Map<User, Map<String, Object>> getAggregated(EmailAggregationKey aggregat
                  * The actual recipients list may differ from the candidates depending on the endpoint properties and the action settings.
                  * The target endpoints properties will determine whether or not each candidate will actually receive an email.
                  */
-                Set<User> users = recipientResolver.recipientUsers(
-                    aggregationKey.getOrgId(),
-                    Stream.concat(
-                        endpoints
-                            .stream()
-                            .map(EndpointRecipientSettings::new),
-                        getActionRecipient(aggregation).stream()
-                    ).collect(Collectors.toSet()),
-                    getSubscribers(eventType, subscribersByEventType)
-                );
+                Set<User> users;
+
+                if (featureFlipper.isEmailConnectorEnabled()) {
+                    users = getSubscribers(eventType, subscribersByEventType).stream().map(username -> {
+                        User usr = new User();
+                        usr.setUsername(username);
+                        return usr;
+                    }).collect(Collectors.toSet());
```

Comment:
Even if we discussed it in Meet, I still dont feel completely comfortable with this change. Theorically, we could skip the aggregation of payloads for some users if user preferences are ignored. Let's keep the old code in this class for now if that's fine with you. It may mean calling IT/RBAC a few extra times but that's not so bad as a temporary solution (until the users service is available).

Reply:
Ok, let's rollback it